plat/rockchip/rk3368/drivers/pmu/pmu.c

```diff
@@ -0,0 +1,436 @@
+/*
+ * Copyright (C) 2016, Fuzhou Rockchip Electronics Co., Ltd.
```

Comment:
Also, note you may optionally add "Rockchip" to the list of contributors in acknowledgements.md.